examples/lens-next-app/src/components/ConnectWalletButton.tsx

```diff
@@ -0,0 +1,12 @@
+import { ConnectKitButton } from "connectkit";
+import { Button } from "./Button";
+
+export const ConnectWalletButton = () => {
```

Comment:
Let's keep a consistent style on these components definition. It seems NextJS docs favour Function Declarations in their examples over Arrow Function Expressions, I would stick to that style.

Reply:
true, I missed it, fixed now!